packages/web/src/scss/tools/_utilities.scss

```diff
@@ -1,9 +1,37 @@
 // Inspired by React UI and Bootstrap
+@use '@tokens' as tokens;
 @use 'sass:list';
 @use 'sass:map';
 @use 'sass:meta';
+@use 'sass:string' as sass-string;
 @use 'string';
 
+@function generate-token-map($tokens-key: null, $prefix: null, $replacement: null, $scope: null) {
+    $result: ();
+    $tokens: if(
+        map.has-key(meta.module-variables(tokens), $tokens-key),
+        map.get(meta.module-variables(tokens), $tokens-key),
+        null
+    );
+
+    @each $key, $values in $tokens {
+        @each $subkey, $value in $values {
+            // Check if the scope is a list, the subkey is not null, and the subkey exists in the scope list.
```

Comment:
Does this mean that it will work if I have only two new tokens? For example, when I need two new accent text colours in my DS, I can introduce just `accent/01/content-basic` and `accent/02/content-basic` (and no backgrouns/borders/subtles...), and it will work. 

We discussed this with Jirka and agreed that this is not an encouraged setup, but we can imagine it will happen.

Reply:
Yes, it should. You can test it on your own. :)